codes/data/usa/merge_policy_and_cases.py

```diff
@@ -1,13 +1,12 @@
 import pandas as pd
 import numpy as np
 import os
-import codes.utils as cutil
 
 from functools import reduce
 
-raw_data_dir = cutil.DATA_RAW / 'usa'
-int_data_dir = cutil.DATA_INTERIM / 'usa'
-proc_data_dir = cutil.DATA_PROCESSED / 'adm1'
+raw_data_dir = '../../../data/raw/usa'
```

Comment:
Just a flag to change back, as mentioned in the conversation tab

Reply:
If you want these to be strings for later in the code, you could always just cast them to strings here

Reply:
ok fixed that on the modified version